tests/console/consoletest_setup.pm

```diff
@@ -57,8 +61,6 @@ sub run {
         assert_script_run 'less /var/log/YaST2/y2log*|grep "Automatic DHCP configuration not started - an interface is already configured"';
     }
 
-    # Stop serial-getty on serial console to avoid serial output pollution with login prompt
-    systemctl "stop serial-getty\@$testapi::serialdev";
     # Mask if is qemu backend as use serial in remote installations e.g. during reboot
     systemctl "mask serial-getty\@$testapi::serialdev" if check_var('BACKEND', 'qemu');
```

Comment:
Why not also move this call?

Reply:
It's already done: https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/9c0cca4be819767949452be797692de7d9671b4b